hw001/hw001.py

```diff
@@ -0,0 +1,85 @@
+# 1. Напишите программу, которая принимает на вход цифру,
+# обозначающую день недели, и проверяет,
+# является ли этот день выходным.
+#  Пример:
+# 6 -> да
+# 7 -> да
+# 1 -> нет
+
+n = int(input('Введите цифру, обозначающую день недели: '))
+if n in range(1, 6):
+    print('Нет')
+elif n in range(6, 8):
+    print('Да')
+else:
+    print('Неверное число!')
+
+
+# 2. Напишите программу для проверки истинности утверждения ¬(X ⋁ Y ⋁ Z) = ¬X ⋀ ¬Y ⋀ ¬Z для всех значений предикат.
```

Comment:
print('X | Y | Z | ¬(X ⋁ Y ⋁ Z) = ¬X ⋀ ¬Y ⋀ ¬Z')
for x in [0, 1]:
    for y in [0, 1 ]:
        for z in [0, 1]:
            left = not (x or y or z)
            right = not x and not y and not z
            print(f'{x} | {y} | {z} | {int(left == right)}')